frontend/src/app/components/rbf-timeline/rbf-timeline.component.ts

```diff
@@ -51,7 +51,7 @@ export class RbfTimelineComponent implements OnInit, OnChanges {
 
   ngOnChanges(changes): void {
     this.rows = this.buildTimelines(this.replacements);
-    if (changes.txid) {
+    if (changes.txid &&!changes.txid.firstChange && changes.txid.previousVaue !== changes.txid.currentValue) {
```

Comment:
Is that a typo? `changes.txid.previousVaue` -> `changes.txid.previousValue`?

Reply:
Fixed it